tools/python_api/src_cpp/pandas/pandas_scan.cpp

```diff
@@ -90,9 +90,12 @@ offset_t tableFunc(TableFuncInput& input, TableFuncOutput& output) {
     }
     auto numValuesToOutput =
         std::min(DEFAULT_VECTOR_CAPACITY, pandasLocalState->end - pandasLocalState->start);
+    auto skips = pandasScanData->getColumnSkips();
```

Comment:
I think we should also skip the bindings the corresponding columns, and also the sniffing.